about.html

```diff
@@ -13,56 +13,85 @@
 
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
+     <header class="top-nav-bar">
+            <div class="top-flex-item"><img src=".\img\lambda-black.png"></div>
+            <div class="top-flex-item"><a href="#">Home</a></div>
```

Comment:
great job creating the Navbar, it appears you forgot to link back to "Home". when the user is in the index.html file they can visit about.html but when they click on "Home" it does nothing since it doesn't link back, which means users can get stuck visiting the site